pyproject.toml

```diff
@@ -27,7 +27,6 @@ exclude = ["pyramid_openapi3/tests/"]
 python = "^3.9"
 
 openapi-core = ">=0.19.0"
-jsonschema-path = ">=0.3.0"
```

Comment:
I included this as it's now imported directly:
https://github.com/Pylons/pyramid_openapi3/blob/fa817cab01d075477a58da7ae24ae4579ebbc9ef/pyramid_openapi3/__init__.py#L8

Is there a specific reason you removed it?

Reply:
It's a dependency of openapi-core, so pyramid_openapi3 does not have to list is as direct dependency. 

In case openapi-core starts using something else, I don't want pyramid_openapi3 to still list is as dependency becase we overlooked removing it. 

I don't have a very strong opinion about it, though. If this is against some best practice, I'm fine with reverting.

Reply:
I've come to the opinion that "if you import it, you should express that in dependencies".

In this specific one case where we only use it to do one thing with `openapi-core`, I guess it's fine to remove. But if we start importing it in any other cases, I think we should have it as a direct dependency.